packages/core/src/App/Components/Layout/Footer/__tests__/network-status.spec.tsx

```diff
@@ -0,0 +1,56 @@
+// TODO refactor old tests in this component
+import React from 'react';
+import { Popover } from '@deriv/components';
+import { NetworkStatus } from '../network-status.jsx';
+import { render, screen } from '@testing-library/react';
+
+const status = {
```

Comment:
do we have types for NetworkStatus? if so, please import them and use for this object

Reply:
No @yauheni-kryzhyk-deriv we don't have type for it